conftest.py

```diff
@@ -23,6 +23,12 @@ def staffuser():
     return UserFactory(short_name="Hello", password='password', is_staff=True)
 
 
+@pytest.fixture()
+def systemuser():
+    # Create it the same way the migration does it
+    return UserFactory(serial='__system__', full_name='System', password='!!')
```

Comment:
Where would that util live?

Would it be a generic `add_user` function that we'd have to use everywhere? (and if so, why not utils for every other model?) Or just an `add_system_user` util?

I'm not entirely convinced this is needed, since it's only in two places that we ever need to create a system user: in a migration and in a test fixture.

Sure, arguably it would ensure that we create them the same way.

However I would say that is wrong too, for a simple reason: if we ever need to change that system user in the database, modifying the `add_system_user` util would **not** be enough, we would need a migration that **modifies** the user.

In fact, in the future if we want to modify that system user, we would:

* create a migration that **modifies** the user in the DB
* modify the code **creating** the user in the tests accordingly
* **not** modify the original migration which **created** the user in the DB

In effect, the two code parts (migration and fixture) are divergent, and I'm not sure it makes sense to try and make them use the same code.